lang/en/admin/server.php

```diff
@@ -36,6 +36,7 @@
     'disabled' => 'Disabled',
     'memory' => 'Memory',
     'memory_limit' => 'Memory Limit',
+    'memory_helper' => 'Wings will add overhead to this value when creating the container. <=2048MB = +10%, <=4069MB = +15%, anything else = +5%',
```

Comment:
I would keep the text neutral. Especially because the overhead can be individually set in the wings config.

Something like:
```suggestion
    'memory_helper' => 'Wings will add overhead to this value when creating the container to make sure it doesn't starve out when using max memory.',
```

Reply:
99.8% of users have no idea it exists... let alone an actual config setting for it.